test/ectest.c

```diff
@@ -1159,6 +1159,43 @@ static int internal_curve_test_method(int n)
     return r;
 }
 
+static int group_field_test(void)
+{
+    int r = 1;
+    BIGNUM *sect521r1_field = NULL;
+    BIGNUM *sect163r2_field = NULL;
+    EC_GROUP *sect521r1_group = NULL;
+    EC_GROUP *sect163r2_group = NULL;
+
+    BN_hex2bn(&sect521r1_field,
+                "01FFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+                "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+                "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+                "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+                "FFFF");
+
+
+    BN_hex2bn(&sect163r2_field,
+                "08000000000000000000000000000000"
+                "00000000C9");
+
+    sect521r1_group = EC_GROUP_new_by_curve_name(NID_secp521r1);
+    if(BN_cmp(sect521r1_field, EC_GROUP_get0_field(sect521r1_group)))
+      r = 0;
+
+    #ifndef OPENSSL_NO_EC2M
```

Comment:
Style nit: We're already inside an ifndef (at the top of the file), so we need a single space indent between "#" and "ifndef"